dev/tests/integration/framework/Magento/TestFramework/Helper/Memory.php

```diff
@@ -46,8 +46,14 @@ public function __construct(\Magento\Framework\Shell $shell)
     public function getRealMemoryUsage()
     {
         $pid = getmypid();
-
-        $result = $this->_getUnixProcessMemoryUsage($pid);
+        try {
+            // try to use the Windows command line
+            // some ports of Unix commands on Windows, such as MinGW, have limited capabilities and cannot be used
+            $result = $this->_getWinProcessMemoryUsage($pid);
```

Comment:
Magento 2.3 technology stack requirements (via https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements-tech.html):
Operating systems (Linux x86-64) - Linux distributions, such as RedHat Enterprise Linux (RHEL), CentOS, Ubuntu, Debian, and similar. Magento is not supported on:

- Windows OS
- Mac OS

So it is kind of funny to see Windows Refs in Application Code itself... 🥇